tests/services/test_encryption_service.py

```
@@ -0,0 +1,120 @@
+        
+        encrypted_file = tmp_path / "test.txt.enc"
+        decrypted_file = tmp_path / "test.txt.dec"
+        
+        file_encryption_service.encrypt_file(db_session, input_file, encrypted_file)
+        
+        # Corrupt one byte of ciphertext (after salt[16] and nonce[12])
+        data = bytearray(encrypted_file.read_bytes())
+        data[30] ^= 0xFF 
+        encrypted_file.write_bytes(data)
+        
+        with pytest.raises(Exception):  # cryptography.exceptions.InvalidTag
```

Ruff (0.15.2)

[warning] Do not assert blind exception: `Exception`

(B017)